typings/index.d.ts

```diff
@@ -0,0 +1,277 @@
+  defaultOptions?: MapOptions;
+  children?: React.ReactNode;
+  width?: number | string;
+  height?: number | string;
+  style?: React.CSSProperties;
+  className?: string;
+}
+
+export interface ObjectManagerFeature {
+  id: number | string;
+  type: 'Feature';
+  geometry: AnyObject;
+  options?: AnyObject;
+  properties?: AnyObject;
+}
+
+export interface ObjectManagerFeatureCollection {
+  type: 'FeatureCollection';
+  features: (ObjectManagerFeatureCollection | ObjectManagerFeature)[];
+}
+
+export type ObjectManagerFeatures =
+  | ObjectManagerFeature
+  | ObjectManagerFeatureCollection
+  | (ObjectManagerFeatureCollection | ObjectManagerFeature)[];
+
+export type ObjectManagerFilter =
+  | string
+  | ((feature: ObjectManagerFeature) => void);
+
+export interface ObjectManagerOptions extends AnyObject {
+  clusterize?: boolean;
+  syncOverlayInit?: boolean;
+  viewportMargin?: number | number[];
+}
+
+/**
+ * TODO: replace `AnyObject` with definitions
+ */
+export interface ObjectManagerObjectsOptions extends AnyObject {}
+export interface ObjectManagerClustersOptions extends AnyObject {}
+
+export interface ObjectManagerProps extends CommonProps {
+  features?: ObjectManagerFeatures;
+  defaultFeatures?: ObjectManagerFeatures;
+  filter?: ObjectManagerFilter;
+  defaultFilter?: ObjectManagerFilter;
+  options?: ObjectManagerOptions;
+  defaultOptions?: ObjectManagerOptions;
+  objects?: ObjectManagerObjectsOptions;
+  defaultObjects?: ObjectManagerObjectsOptions;
+  clusters?: ObjectManagerClustersOptions;
+  defaultClusters?: ObjectManagerClustersOptions;
+}
+
+export interface ClustererOptions extends AnyObject {
+  gridSize?: number;
+  groupByCoordinates?: boolean;
+  hasBalloon?: boolean;
+  hasHint?: boolean;
+  margin?: number | number[] | number[][];
+  maxZoom?: number | number[];
+  minClusterSize?: number;
+  preset?: string;
+  showInAlphabeticalOrder?: boolean;
+  useMapMargin?: boolean;
+  viewportMargin?: number | number[] | number[][];
+  zoomMargin?: number | number[] | number[][];
+}
+
+export interface ClustererProps extends CommonProps {
+  options?: ClustererOptions;
+  defaultOptions?: ClustererOptions;
+}
+
+export interface PanoramaOptions extends AnyObject {
+  autoFitToViewport?: 'none' | 'ifNull' | 'always';
+  controls?: string[];
+  direction?: string | number[];
+  hotkeysEnabled?: boolean;
+  scrollZoomBehavior?: boolean;
+  span?: string | number[];
+  suppressMapOpenBlock?: boolean;
+}
+
+export interface PanoramaProps extends CommonProps {
+  coordinates?: number[];
+  defaultCoordinates?: number[];
+  options?: PanoramaOptions;
+  defaultOptions?: PanoramaOptions;
+  children?: React.ReactNode;
+  width?: number | string;
+  height?: number | string;
+  style?: React.CSSProperties;
+  className?: string;
+}
+
+export interface WithYMapsProps {
+  ymaps: YMapsApi;
+}
+
+export function withYMaps<P>(
+  component: React.ComponentType<P>,
+  waitForApi?: boolean,
+  modules?: string
```

Comment:
@zalishchuk I suppose it should be array?

Reply:
Yes, thanks, fixed in fae8c2a.